packages/cadl-rlc-test/package.json

```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "type": "module",
   "dependencies": {
-    "@cadl-lang/compiler": "~0.40.0",
+    "@typespec/compiler": "~0.41.0",
     "@azure-tools/cadl-typescript": "1.0.0-beta.10",
-    "@cadl-lang/rest": "~0.40.0",
-    "@cadl-lang/openapi": "~0.40.0",
-    "@cadl-lang/versioning": "~0.40.0",
-    "@azure-tools/cadl-azure-core": "~0.26.0",
-    "@azure-tools/cadl-autorest": "~0.26.0",
-    "@azure-tools/cadl-dpg": "~0.26.0",
```

Comment:
Just curious, why we still have denpendency for cadl-dpg?
https://github.com/Azure/autorest.typescript/blob/34637ea97bf56924e555e5e88084940a56fb5948/common/config/rush/pnpm-lock.yaml#L159